addons/survey/models/survey_survey_template.py

```diff
@@ -0,0 +1,281 @@
+                    'question_type': 'char_box',
+                    'save_as_nickname': True
+                }),
+                (0, 0, { # survey.question
+                    'title': _('Quiz'),
+                    'is_page': True,
+                    'question_type': False
+                }),
+                (0, 0, { # survey.question
+                    'title': _('In which country did the bonsai technique develop?'),
+                    'question_type': 'simple_choice',
+                    'suggested_answer_ids': [
+                        (0, 0, { # survey.question.answer
+                            'value': _('Japan'),
+                            'is_correct': True,
+                            'answer_score': 20
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('China')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Vietnam')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('South Korea')
+                        })
+                    ]
+                }),
+                (0, 0, { # survey.question
+                    'title': _('In the list below, select all the coniferous.'),
+                    'question_type': 'multiple_choice',
+                    'suggested_answer_ids': [
+                        (0, 0, { # survey.question.answer
+                            'value': _('Douglas Fir'),
+                            'is_correct': True,
+                            'answer_score': 5
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Norway Spruce'),
+                            'is_correct': True,
+                            'answer_score': 5
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('European Yew'),
+                            'is_correct': True,
+                            'answer_score': 5
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Mountain Pine'),
+                            'is_correct': True,
+                            'answer_score': 5
+                        })
+                    ]
+                })
+            ]
+        }))
+
+    def action_show_sample(self, survey):
+        action = self.env['ir.actions.act_window']._for_xml_id('survey.action_survey_form')
+        action['target'] = 'main'
+        action['views'] = [[self.env.ref('survey.survey_survey_view_form').id, 'form']]
+        action['res_id'] = survey.id
+        action['context'] = dict(ast.literal_eval(action.get('context', {})),
+            form_view_initial_mode='edit',
+            create=False
+        )
```

Comment:
As you define multiple keys, it's a bit easier to use the "update" method on the dict and give it your keys.

Reply:
I meant to update the "action" variable :smile: 

But keep it like that, I had a small discussion with Thibault and he doesn't agree with me, a matter of taste I suppose.